include/rpm/rpmfileutil.h

```diff
@@ -107,14 +107,6 @@ char * rpmGenPath	(const char * urlroot,
  */
 char * rpmGetPath (const char * path, ...) RPM_GNUC_NULL_TERMINATED;
 
-/** \ingroup rpmfileutil
- * Check whether pattern contains any glob metacharacters.
- * @param pattern	glob pattern
- * @param quote		allow backslash quoting of metacharacters?
- * @return		1 if pattern contains globs, 0 otherwise
- */
-int rpmIsGlob(const char * pattern, int quote);
```

Comment:
rpmIsGlob() is part of the public API so care needs to be taken when removing. As it happens, we'll be bumping the soname in 4.19 so just now removing it is okay (I doubt it's being used outside rpm anyway), but its removal does need to be documented in the commit message.

Reply:
Oh-oh, I knew I had forgotten something :sweat_smile: Thanks for noticing, will fix.

Reply:
Fixed now.

Reply:
Hmm, I still see no mention of it? :monocle_face:

Reply:
Sorry, I somehow missed the fact that this was about the `rpmIsGlob()` and not `rpmGlob()` declaration, I updated the latter only :facepalm: 

And thanks for noticing, based on where we are in the 4.18 cycle, I guess we're not going to include these commits in it anyway, so in that case, the planned 4.19 bump is a good opportunity for this, as well as for the subtle `rpmGlob()` change, indeed.

Reply:
... and hopefully fixed now, for real.